leader-election-impl/src/main/java/com/palantir/paxos/PaxosQuorumChecker.java

```diff
@@ -236,7 +286,7 @@ private PaxosQuorumChecker() {
             }
         }, OUTSTANDING_REQUEST_CANCELLATION_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS);
 
-        if (log.isDebugEnabled() && shouldLogCancellationStatus()) {
+        if (log.isDebugEnabled() && shouldLogException()) {
```

Comment:
nit: this one isn't really an exception though. `shouldLogDiagnosticInformation()` maybe?